### ISSUE TITLE
Dananshan Miao

### ISSUE THREAD

**Issue by MuffinLinwist** [glottolog/glottolog#1089]
I'm opening an issue regarding the classification of a language in the [Chuanqiandian (8)](https://glottolog.org/resource/languoid/id/chua1248) cluster, as suggested in [lexibank/joophonosemantic](https://github.com/lexibank/joophonosemantic/issues/3):

`The standard dialect of Chuanqiandian Cluster Miao is the variety spoken in Dananshan (Bijie, Guizhou)and all the Chinese sources I have consulted made reference to this variety. Glottolog does not seem to have a particular page for Dananshan Miao.`

Perhaps, @ianjoo or @LinguList can provide more info about it?

**Comment by d97hah** [comment]
The issue is the following:
- The Chinese linguists use the name Chuanqiandian (CQD) for both a fangyan
and a cifangyan (see Niederer 1998:51 for the simplest summary). Normally
those words correspond to 'dialect' and 'subdialect' but in this case the
members of the fangyan are not intelligible so perhaps a closer
correspondence is 'subgroup' and 'language'.
- The Ethnologue [cqd] entry contains (some, but not all) information
(locations, dialect listing and speaker number) that pertains to the
subgroup
interpretation of CQD. However, other members of the CQD subgroup, e.g.
Ge [hmj], Luopohe Hmong [hml], Diandongbei-Large Flowery Miao [hmd], already
have separate entries in Ethnologue, as do some varieties in the CQD
cifangyan (language?) Horned Miao [hrm], Sinicized Miao [hmz], Hmong Daw
[mww],
Hmong Njua [hnj] etc. These entries, judging from their name and location,
however, do not exhaust the CQD-cifangyan varieties known, but most (all?)
of
the CQD-cifangyan varieties are said to be intelligible, or at least not
clearly the opposite. I therefore considered Ethnologue's [cqd] entry, as
defined, spurious and removed it.
- At the same time I introduced a new language level entry for Northern Hua
Miao
given a clear unintelligibility claim by McLaughlin (2012:9-10).
- So where does CQD Dananshan and other known varieties fit in? They could
be
with Northern Hua Miao, almost certainly the same language by
intelligibility. Alternatively,
they could be with the mysterious Horned Miao [hrm], which is only really
defined in an
unpublished manuscript by Johnson 1998 (see Castro et al. 2012:6,
McLaughlin 2012:9,
Hattaway 2000:215). The locations of Horned Miao [hrm] overlap with other
CQD varieties
 and it can't be said which data represent it without seeing Johnson 1998
(which I haven't).
- It might just be simpler for everybody, and equally right/wrong, to
reinstate the [cqd] entry
with a narrower interpretation in place of the Northern Hua Miao entry, so
I will do that for
next version.

The TL/DR for you is: connect the CQD Dananshan data with nort3244 if you
want to be compliant with Glottolog 5.0 and
with chua1256 if you want to attach at the language-level, for 5.1.

all the best, H

Pada Jum, 2 Agu 2024 pukul 17.31 C. M. B. Ugarte ***@***.***>
menulis:
